### DISCUSSION TITLE
Loading progress while deployed on Netlify?

### DISCUSSION THREAD

**Discussion by preyneri** [jeffreylanters/react-unity-webgl#221] [Help Wanted!]
Hello,

First of all, I'd like to say what an amazing project this is, and how fortunate I am to have discovered it! I have just one small issue thus far. 

I'm using Gatsby deployed to Netlify. In my local development, the progress value from the example implementation displays correctly. However, when I deploy the site to Netlify, the progress value remains at zero until the game appears. I am uploading uncompressed builds.

I know this may be a bit more of a Netlify-specific issue, but I thought I'd ask here to see if this problem sounds familiar or spurs any ideas.

**Comment by jeffreylanters** [comment]
Hi! You're welcome. I'm sorry, I'm receiving questions about these kinds of platforms more ofter. I've never used Gatsby or Netlify before, so I'm afraid I can't help you. Can I see the behaviour somewhere, is your site live (and public)? I might be able to debug it that way. But I would not be surprised if this was any issue within the Unity code base.
